Makefile

```diff
@@ -190,10 +175,14 @@ GENERIC_RUN_PROFILE=generic-profile
 autoconfig:
 	$(call run-no-test,autoconfig,$(GENERIC_RUN_PROFILE),$(DOMAIN))
 
+.PHONY: results
+results:
+	xdg-open logic-inspector/index.html &> /dev/null || open logic-inspector/index.html &> /dev/null
```

Comment:
no change needed:  xdg-open sadly does not work particularly well on my system, and the manpages aren't super helpful, but it's clear you made the right call with xdg-open because there definitely don't seem to be better options!  (Well, we could hardcode firefox, but that is its own headaches.)